uint32 fabric_id # fabric index
SegmentState[] states